Ducky-Chickens/bangazonCLI
# Tm customer revenue

# Description
Provides capability in given prompt to view active customers accrued revenue (both by customer products in order & overall accrued revenue).

## Related Ticket(s)
Issue Ticket: #10 

## Problem to Solve
Gives UI capabilities to add new products per active customer


## Steps to Test Adding Customer Products
+ Run `db:reset`
+ Run `npm i -g`
+ Run `bangazon`
+ Select option `2` to activate a customer of choosing
+ Select option `11` upon returning to the main prompt to begin adding a product
+ CLI should display table of revenue in terminal (i.e., with active customer `8` -- Ben Gentle)
```bash
Order#4
----------------------------------------------------
Fantastic Wooden Shoes          1          $379

Order#10
----------------------------------------------------
Refined Fresh Tuna              1          $39

Order#13
----------------------------------------------------
Ergonomic Frozen Chicken        1          $126

Order#14
----------------------------------------------------
Ergonomic Frozen Chicken        1          $126

Total revenue: $670
```

## Testing


**To Run Tests**:
+ Run `npm test`
+ View tests in `getCustomerRevenue.test.js`, returning array of products per selected `customer_id` and testing for the appropriate count of customer products from `order_products`
